docker-compose.yml

```diff
@@ -24,6 +24,7 @@ services:
       - "CLOUDCONCIERGE_PULLREVIEWERS=$CLOUDCONCIERGE_PullReviewers"
       - "CLOUDCONCIERGE_RESOURCESWHITELIST=$CLOUDCONCIERGE_ResourcesWhiteList"
       - "CLOUDCONCIERGE_WORKSPACEDIRECTORIES=$CLOUDCONCIERGE_WorkspaceDirectories"
+      - "CLOUDCONCIERGE_CLOUDREGIONS=$CLOUDCONCIERGE_CloudRegions"
```

Comment:
Please make corresponding updates to the example configuration files, and add comments noting that how for now only one region per provider is allowed.